api-interop-layer/data/index.js

```diff
@@ -12,90 +12,115 @@ export const getDataForPoint = async (lat, lon) => {
   logger.verbose(`fetching forecast for ${lat}, ${lon}}`);
   const { point, place, grid } = await getPoint(lat, lon);
 
-  const satellitePromise = getSatellite({ grid });
-
-  const { forecast, observed } = await Promise.all([
-    getForecast({ grid, place }),
-    getObservations({ grid, point }),
-  ]).then(([forecastData, obsData]) => {
-    // The forecast endpoint returns extra information about the grid. Why? I
-    // dunno. But anyway, let's put it with the other grid info and remove it
-    // from the forecast data.
-    Object.assign(grid, forecastData.gridData);
-    delete forecastData.gridData;
-
-    // The daily forecast endpoint includes elevation information, but that
-    // really belongs with the grid, so let's move that, too.
-    if (forecastData.daily.elevation) {
-      grid.elevation = forecastData.daily.elevation;
-      delete forecastData.daily.elevation;
-    }
+  let satellitePromise = Promise.resolve({ error: true });
+  let forecast = { daily: { error: true } };
+  let observed = { error: true };
```

Comment:
Bit confused as to why we're setting them to true here by default. Are these values overwritten later on?

Reply:
Yes, they get overwritten if there is grid data. They may get overwritten with the exact same thing if fetching them causes any errors. I thought about using an `Object.assign` kind of thing and chose not to, but I don't recall why.